tests/Predis/Pipeline/PipelineTest.php

```diff
@@ -492,7 +493,13 @@ public function testIntegrationWithServerExceptionInCallableBlock(): void
             // NOOP
         }
 
-        $this->assertInstanceOf('Predis\Response\ServerException', $exception);
+        // TODO: check and change the name if needed
+        if($client->getConnection() instanceof RelayConnection) {
+            $this->assertInstanceOf('Relay\Exception', $exception);
```

Comment:
The `executePipeline()` needs to catch and convert exceptions. This test must remain unchanged.